docs/learn/learn-account-balances.md

```diff
@@ -97,24 +97,23 @@ Free: 20 DOT
 Frozen (locked): 20 DOT
 Reserved (held): 80 DOT
 Spendable: 19 DOT (Free - ED)
-Untouchable: 21 DOT (ED)
+Untouchable: 1 DOT (ED)
 ```
 
 ![balance-example-4](../assets/balance-example-4.png)
 
-Note how, through the fungible trait, the system uses the reserved balance. In other words, locks are
-subtracted from the free balance but overlap with the reserved balance, and with themselves (see below). The free and reserved portions remain 20 DOT and 80 DOT, respectively. We also have 20 DOT as a frozen balance because of the governance lock, and the untouchable balance would thus be 21 DOT.
+Note how, through the fungible trait, the system uses the reserved balance. Locks are
+imposed on the total balance, they overlap with the reserved balance, and with themselves (see below). The free and reserved portions remain 20 DOT and 80 DOT, respectively. We also have 20 DOT as a frozen balance because of the governance lock. The untouchable balance remains 1 DOT as it is defined as the frozen balance in excess of any holds (in this case frozen < holds, so only the ED is untouchable).
 
-A vested transfer of 50 DOT will
-grow the frozen balance to 50 DOT and the untouchable balance to 51 DOT, using the already frozen 20 DOT plus an additional 30 DOT from the reserved balance without touching the free and spendable balances.
+If the account votes on another governance proposal with 85 DOT (assuming the previous proposal is still ongoing or that it ended on the winning side with a lock), the frozen balance will increase to 85 DOT and the untouchable balance to 6 DOT, using the already frozen 20 DOT plus an additional 60 DOT from the reserved balance and 5 DOT from the free balance, in excess of the reserved balance.
 
 
 ```
 Free: 20 DOT
-Frozen (locked): 50 DOT
+Frozen (locked): 85 DOT
 Reserved (held): 80 DOT
-Spendable: 19 DOT (Free - ED)
-Untouchable: 51 DOT (ED)
+Spendable: 14 DOT (Free - untouchable)
+Untouchable: 6 DOT (ED + frozen in excess of reserved)
```

Comment:
It is:
Untouchable: 5
Spendable: 15

Remember the visual aid how frozen overlaps with ED and untouchable is only the excess from the holds.